zerver/webhooks/trello/doc.md

```diff
@@ -1,17 +1,19 @@
-!!! tip ""
-
-    Note that [Zapier][1] is usually a simpler way to
-    integrate Trello with Zulip.
+# Zulip Trello integration
 
 Get Zulip notifications from your Trello boards!
 
-[1]: ./zapier
+!!! warn ""
```

Comment:
Tip -> Warning

Reply:
I think it's better as a tip, so switched it back.